python/GafferUI/CompoundEditor.py

```diff
@@ -1431,19 +1431,18 @@ def _restoreWindowState( gafferWindow, boundData ) :
 			targetScreen = screens[ boundData["screen"] ]
 			window.setScreen( targetScreen )
 
-	screenGeom = targetScreen.availableGeometry()
-	window.setGeometry(
-		( boundData["bound"].min()[0] * screenGeom.width() ) + screenGeom.x(),
-		( ( 1.0 - boundData["bound"].max()[1] ) * screenGeom.height() ) + screenGeom.y(),
-		( boundData["bound"].size()[0] * screenGeom.width() ),
-		( boundData["bound"].size()[1] * screenGeom.height() )
-	)
-
 	if boundData["fullScreen"] :
 		window.setWindowState( QtCore.Qt.WindowFullScreen )
 	elif boundData["maximized"] and sys.platform != "darwin" :
 		window.setWindowState( QtCore.Qt.WindowMaximized )
 	else :
+		screenGeom = targetScreen.availableGeometry()
```

Comment:
This makes sense to me, but I think it leaves [L1417-1424](https://github.com/GafferHQ/gaffer/pull/5118/files#diff-eefb666c4db44f7ab31b33df0ab4d76c82e04a7a4e6f3787caf4805681a09882R1417-R1424) looking redundant. I tried tidying this up in https://github.com/johnhaddon/gaffer/commit/968a50fbd752e0df25f3b8fdf6d23e8c9e4f553a, and it seems to work - does that make sense to you?

Reply:
Ah yes, that does make sense. And it keeps Windows windows happy as well.